SwiftCompilerSources/Sources/SIL/Type.swift

```diff
@@ -224,6 +224,10 @@ public struct Type : TypeProperties, CustomStringConvertible, NoReflectionChildr
     }
     return false
   }
+
+  public func mapOutOfContext(in function: Function) -> Type {
```

Comment:
This is now called `mapOutOfEnvironment`

Reply:
Thanks, adopted new naming for similar functions I've added in this PR